miner/worker.go

```diff
@@ -81,6 +83,14 @@ const (
 	staleThreshold = 7
 )
 
+var (
+	// Metrics for the skipped txs
+	gasLimitExceededMeter       = metrics.NewRegisteredMeter("miner/skipped_txs/gas_limit_exceeded", nil)
```

Comment:
can use `Counter`. It's a bit weird to use `Meter`, because `Meter` is for rate monitoring, and skipping transactions due to these reasons isn't a common case.

A `Counter` example: https://github.com/scroll-tech/go-ethereum/blob/v4.3.0/core/state/snapshot/generate.go#L81